data/lang/ui-core/en.json

```diff
@@ -63,6 +63,10 @@
     "description": "Automatically find shortest jump route (measured in time) in star map",
     "message": "Auto Route"
   },
+  "AUTOMATED_MESSAGE": {
+    "description": "For comm messages from bots",
+    "message": "AUTOMATED MESSAGE"
```

Comment:
` we should probably change that translation string to refer to "SystemNet" or something like that, since we've somewhat settled on a lack of interstellar comms :D`

@sturnclaw Sorry, I didn't get this. Is this the string you meant?

Reply:
No - was referring to the description for the Computers commodity item, sorry. Bit of cross-talk there on my end!

Reply:
Aight!